e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -17,6 +17,13 @@ param(
     [Parameter()]
     [switch] $InstallDependencies,
 
+    # Obsoleted. Recommend docker on WSL2 instead - setup with <repo>/e2e/test/docker/haproxy.sh on WSL.
```

Comment:
Adding this flag instead of removing all the docker on windows setup to allow existing dev to continue to run proxy test on docker for Windows.

Reply:
Changed to Deprecated